docs/guides/specify-runtime-options.mdx

```diff
@@ -138,10 +138,15 @@ estimator.options.resilience.zne_mitigation = True
 
 The `SamplerV2.run` method accepts two arguments: a list of PUBs, each of which can specify a PUB-specific value for shots, and a shots keyword argument. These shot values are a part of the Sampler execution interface, and are independent of the Runtime Sampler's options.  They take precedence over any values specified as options in order to comply with the Sampler abstraction.
 
-However, if `shots` is not specified by any PUB or in the run keyword argument (equivalently, if they are all `None`), then the shots value from the options is used, most notably `default_shots`. To summarize, this is the order of precedence for specifying shots in the Sampler, for any particular PUB:
+However, if `shots` is not specified by any PUB or in the run keyword argument ( or if they are all `None`), then the shots value from the options is used, most notably `default_shots`.
+
+Finally, if the `twirling` options `num_randomizations` or `shots_per_randomization` are specified, those values take precedence over `default_shots`  (but not `shots` values specified as part of the `run` call).
+
+To summarize, this is the order of precedence for specifying shots in the Sampler, for any particular PUB:
 
 1. If the PUB specifies shots, use that value.
-2. If the shots keyword argument is specified in `run`, use that value.
+2. If the `shots` keyword argument is specified in `run`, use that value.
+3. If `num_randomizations` or `shots_per_randomization` are specified as twirling options, use those values.
```

Comment:
I would reword it to specify that these `twirling` options are enabled by default so if `default_shots` are the only way shots are set then the number of shots will be:

the product of ``num_randomizations`` and ``shots_per_randomization``

Maybe we can also link the twirling options (https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions) - Jessie's new updates (https://github.com/Qiskit/qiskit-ibm-runtime/pull/1884) aren't live yet but will be on next release